test/docker-in-docker/test-scripts/docker-test-init.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+retry_count=0
```

Comment:
It would be good to add a comment saying this needs to be kept up-to-date with the retry logic in the install.sh script. A future improvement would be to pull this out into a helper lib that can be used within the install.sh and invoked from the test as well to reduce duplication and ensure the test is exercising the correct logic.

Reply:
Makes sense, I will shortly create a follow up PR -- avoiding another commit to this PR, else it would dismiss your approval 🤦‍♀️